app/subscriber/src/features/search-page/utils/getFilterActions.ts

```diff
@@ -0,0 +1,32 @@
+import { ActionName, IActionModel, IFilterActionSettingsModel } from 'tno-core';
+
```

Comment:
Util method to retrieve what would be the stored state of an action when utilized.  Avoids duplication when creating an action to store